app/routes/bdb/components/BdbPage.tsx

```diff
@@ -174,6 +176,20 @@ const BdbPage = () => {
       </Card>
 
       <Flex width="fit-content">
+        <button
+          onClick={() =>
+            setQuery({
+              ...query,
+              semester: getSemesterSlugOffset(
+                currentCompanySemester!.id,
+                companySemesters,
+                -1,
+              ),
+            })
+          }
+        >
+          <Icon iconNode={<ChevronLeft />} />
+        </button>
```

Comment:
```suggestion
        <Icon iconNode={<ChevronLeft />} onPress={() =>
            setQuery({
              ...query,
              semester: getSemesterSlugOffset(
                currentCompanySemester!.id,
                companySemesters,
                -1,
              ),
            })
          }
         />
        </button>
```

Just add the onPress directly to the icon